.github/renovate.json

```diff
@@ -27,7 +27,9 @@
       "matchDepTypes": ["dependencies", "devDependencies", "peerDependencies"],
       "excludePackagePatterns":[
         "^@angular",
```

Comment:
Do we already have another process for manually updating arrow? If not, think it'd make sense to do it manually as part of https://github.com/ni/nimble/issues/1747 alongside the pinned playwright deps? If so I can update those instructions.

Reply:
Nuget doesn't have a peer dependency concept as far as I'm aware so maybe it's possible for the nimble-blazor and blazor app arrow versions to be out of sync? Not sure how assemblies resolve, can multiple different versions resolve in the same C# app? Either way I wouldn't expect any of that from keeping oour dependency up to date so yea lets have a manual task

Reply:
Updated.